package.json

```diff
@@ -235,20 +235,21 @@
     "@storybook/addons": "^5.3.19",
     "@storybook/preset-create-react-app": "^3.1.4",
     "@storybook/react": "^5.3.19",
-    "@testing-library/jest-dom": "5.11.4",
+    "@testing-library/jest-dom": "5.11.5",
     "@testing-library/react": "10.4.9",
     "@typechain/web3-v1": "^1.0.0",
     "@types/history": "4.6.2",
-    "@types/jest": "^26.0.14",
+    "@types/jest": "^26.0.15",
     "@types/lodash.memoize": "^4.1.6",
-    "@types/node": "^14.11.8",
-    "@types/react": "^16.9.52",
-    "@types/react-dom": "^16.9.6",
+    "@types/node": "^14.14.5",
+    "@types/react": "^16.9.53",
+    "@types/react-dom": "^16.9.8",
     "@types/react-redux": "^7.1.9",
     "@types/react-router-dom": "^5.1.6",
     "@types/styled-components": "^5.1.4",
-    "@typescript-eslint/eslint-plugin": "3.9.1",
-    "@typescript-eslint/parser": "3.9.1",
+    "@typescript-eslint/eslint-plugin": "3.10.1",
+    "@typescript-eslint/parser": "3.10.1",
+    "@typescript-eslint/scope-manager": "3.10.1",
```

Comment:
why is this needed? is it new, right?

Reply:
Yes, I was testing react-scripts v4 that it's a bit unstable yet, but it required it. I can remove it while we wait to migrate to latest react-scripts

Reply:
My two cents, if it won't be needed for this dep bump, let's remove it and add it back when we definitely upgrade `react-scripts`

Reply:
Done